crates/core_arch/src/wasm32/mod.rs

```diff
@@ -12,7 +12,7 @@ mod simd128;
 pub use self::simd128::*;
 
 mod relaxed_simd;
-#[unstable(feature = "stdarch_wasm_relaxed_simd", issue = "111196")]
+#[stable(feature = "stdarch_wasm_relaxed_simd", since = "CURRENT_RUSTC_VERSION")]
```

Comment:
Is `CURRENT_RUSTC_VERSION` how it's done in `stdarch` as well?

Reply:
No, you have to put in the actual version. Since stdarch is a submodule it doesn't get updated by the release time.

Reply:
Alright, will do that after the FCP in https://github.com/rust-lang/rust/pull/117468.